viewing_party/party.py

```diff
@@ -1,23 +1,170 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    if title is None or genre is None or rating is None:
+        return None
+    return new_movie

```

Comment:
So if we look at the directions in the README we see that we need to return  `None` if: 

If `title` is falsy, `genre` is falsy, or `rating` is falsy, this function should return `None`

Right now your function is only returning `None` if the values of the arguments of your function are `None` but what happens if you are one of the values is an empty string or `0`? How could we change your function to consider these questions?